src/api/coreControl/peripheralDevices.ts

```diff
@@ -12,11 +12,11 @@ export interface SubDeviceStatePayload {
 	desiredSubDeviceState: SofieSubDeviceState
 }
 
-export function isSofieSubDeviceState(obj: any): obj is SofieSubDeviceState {
+export function isSofieSubDeviceState(obj: SofieSubDeviceState): obj is SofieSubDeviceState {
```

Comment:
What was the reason for changing these?
I thought the purpose of these functions were to safely cast something of any type to `SofieSubDeviceState`, if the input already has to be `SofieSubDeviceState` does that mean having to do the cast at each call site before using this to verify the cast?

Reply:
There's a linter error with `any`: `Argument 'obj' should be typed with a non-any type.`
I've changed it to `Record<string, any>`